### PR TITLE
fix: NFT modal deposit address display

### LINKED ISSUE #467: NFT Modal: deposit address empty not prefilled with CF ID

### PR DESCRIPTION
**Pull request by charleenfei**
Please review https://github.com/centrifuge/gateway/pull/463 first!

Closes #467 

Screenshot 2020-06-11 at 14 57 18

### CODE REVIEW

File: packages/ui/src/documents/__tests__/MintNftForm.test.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ describe('Mint NFT Form', () => {
     );
```
Comment (ilinzweilin) [inline packages/ui/src/documents/__tests__/MintNftForm.test.tsx:51]: I don't think you need to pass the user here anymore

File: packages/ui/src/documents/__tests__/MintNftForm.test.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ describe('Mint NFT Form', () => {
     expect(onSubmit).toHaveBeenCalledTimes(0);
   });
 
-  it('Should  submit the form', async () => {
+  it('Should submit the form', async () => {
```
Comment (ilinzweilin) [inline packages/ui/src/documents/__tests__/MintNftForm.test.tsx:74]: same user prop not needed anymore

**Review by ilinzweilin — APPROVED** [review]

File: packages/ui/src/documents/__tests__/MintNftForm.test.tsx (hunk 1 above)
Comment (charleenfei) [reply packages/ui/src/documents/__tests__/MintNftForm.test.tsx:51]: 👍

File: packages/ui/src/documents/__tests__/MintNftForm.test.tsx (hunk 2 above)
Comment (charleenfei) [reply packages/ui/src/documents/__tests__/MintNftForm.test.tsx:74]: removed

**Pull request merged**